src/exploration/manuscript/code/get_paper_csvs.ipynb

```diff
@@ -0,0 +1,675 @@
+{
```

Comment:
I will still leave the subfolder because one notebook generates images in a different subfolder (manuscript/figures), but I am adding some documentation to the notebooks.